### PR TITLE
F/511 reformat line chart

### LINKED ISSUE #511: reformat line charts
**Is your feature request related to a problem? Please describe:**  
The line chart layout needs an upgrade.

**Please describe the solution you'd like:**  
Here are some temporarily requirements, (which can be extended):
- shrink chart height
- add background color to comparison line chart (database color)
- move tile headers (database, title, metric details)
- remove x axis label (?)

**Affected Component(s):**  
`Frontend`

### PR DESCRIPTION
**Pull request by MaxSchneider1337**
# Resolves #511

**Does your pull request solve a problem? Please describe:**  
improve design for the linecharts.

**Does your pull request add a feature? Please describe:**  
zero

**Affected Component(s):**  
Frontend, Linechart, Tile 

**Additional context:**  
zero

### CODE REVIEW

**Comment by MaxSchneider1337** [comment]
this is how it looks like
image

File: hyrisecockpit/frontend/src/components/charts/Linechart.vue [hunk 1]
```diff
@@ -158,6 +156,7 @@ function useLineChartConfiguration(
     return {
       y: data,
       mode: "lines+markers",
+      fill: "tonexty",
```
Comment (schTi) [inline hyrisecockpit/frontend/src/components/charts/Linechart.vue]: Color should only be filled on comparison page, right?

**Review by schTi — CHANGES_REQUESTED** [review]

File: hyrisecockpit/frontend/src/components/charts/Linechart.vue (hunk 1 above)
Comment (MaxSchneider1337) [reply hyrisecockpit/frontend/src/components/charts/Linechart.vue]: looks great on overview as well

Comment (schTi) [reply hyrisecockpit/frontend/src/components/charts/Linechart.vue]: Okay, can you add a screen shot of it?

Comment (Bouncner) [reply hyrisecockpit/frontend/src/components/charts/Linechart.vue]: And can you please check how that looks with more than 3 databases? Sample images from anywhere would be sufficient for me.

Comment (MaxSchneider1337) [reply hyrisecockpit/frontend/src/components/charts/Linechart.vue]: compare: 
image

overview: 
image

Comment (MaxSchneider1337) [reply hyrisecockpit/frontend/src/components/charts/Linechart.vue]: image
4 dbs break the format, but the plots seem to be too small anyway. I would suggest limiting the number of dbs on the comparison site to 3.

Comment (Bouncner) [reply hyrisecockpit/frontend/src/components/charts/Linechart.vue]: I agree that 4 is quite a lot, but the formatting should not be an issue with "a responsive" design, shouldn't it?

For the overview, I think the default line chart is a better fit.

Comment (MaxSchneider1337) [reply hyrisecockpit/frontend/src/components/charts/Linechart.vue]: 4 dbs also breaks the system info
image

Comment (MaxSchneider1337) [reply hyrisecockpit/frontend/src/components/charts/Linechart.vue]: @Bouncner sadly even most responsive designs break on a minimal with. We could also limit the max dbs according to the screen with. There should be enough space on a 4k screen, for 4 dbs.

Comment (Bouncner) [reply hyrisecockpit/frontend/src/components/charts/Linechart.vue]: I don't expect you to fit 17 databases into a single row. But the last picture of the instance view showed a case which should be managable when the database button, the title, and the metric each get their own row (or 2 and 1 on two rows). But that's not super important.

**Review by schTi — APPROVED** [review]
I am still not a fan of the idea to move the metric title to the left, but it is okay for me now. Nice design upgrade in general! :fire:

File: hyrisecockpit/frontend/src/components/charts/Linechart.vue (hunk 1 above)
Comment (MaxSchneider1337) [reply hyrisecockpit/frontend/src/components/charts/Linechart.vue]: @Bouncner I wrote an extra issue for that. I tried to implement a simple responsive design, but it seems to be a bit more complex.

**Pull request merged**